lang/camlp5/Portfile

```diff
@@ -28,12 +28,13 @@ checksums           rmd160  459a703b56523f907f0c0c8b9ebf98f11edad676 \
                     size    1284440
 
 depends_lib       port:ocaml \
-    port:ocaml-bos \
-    port:ocaml-re \
+    port:ocaml-findlib \
     port:ocaml-camlp-streams \
-    port:ocaml-ounit \
-    port:ocaml-pcre2 \
+    bin:perl:perl5 \
```

Comment:
Is perl a _lib_ dependency or rather a build/run dependency?

Reply:
> Is perl a _lib_ dependency or rather a build/run dependency?

You are right, I'll amend.